github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,28 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
+
+s = slack_client("slack_conn")
+
+
+def on_schedule() -> None:
```

Comment:
Just making sure: schedular-based handler functions shouldn't have a parameter, like `event` in connection-based handlers?

Reply:
I will check

Reply:
```
[plan] trigger "github_copilot_seats:connection:github_copilot_seats/slack_conn/check_now": not as desired, will update
Error: action update_trigger github_copilot_seats:schedule:@daily/check_daily: update schedule: schedule: update scheduler workflow: workflow not found for ID: temporal-sys-scheduler:trg_01jg5aj8mqfnetdc0wtjqjnh83
```

This is the error you get if you try to add an event parameter.

Reply:
Not sure I understand this error message - I was asking if the Python function that handles the scheduler trigger needs to have an `event` parameter like regular trigger entry points, or if it's fine without it like in your code.

But I'm assuming you tested triggering these events now, and if that works it answers my question.